02.calendar/cal.rb

```diff
@@ -4,7 +4,7 @@
 
 def main
   year, month = get_year_month
-  puts "#{month}月 #{year}".rjust(8 + month.digits.length + year.digits.length)
+  puts "#{month}月 #{year}".center(15 + month.digits.length + year.digits.length)
```

Comment:
```suggestion
  puts "#{month}月 #{year}".center(??)
```

カレンダーの横幅は静的な値として決められると思うので、「これ」という数値を指定しましょう。